### PR TITLE
Objective-C tracker v1.0.0 release

### PR DESCRIPTION
**Pull request by mhadam**

### CODE REVIEW

File: _posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+title: "Snowplow Objective-C Tracker 1.0.0 released"
+title-short: Snowplow Objective-C Tracker 1.0.0
+tags: [snowplow, mobile, obj-c, ios]
+author: Mike
+category: Releases
+permalink: /blog/2019/02/20/snowplow-ios-tracker-1.0.0-released/
+discourse: true
+---
+
+We are pleased to announce a new release of the [Snowplow Objective-C Tracker][ios-tracker]. [Version 1.0.0][tracker-100] brings dependency updates, some bug fixes and support for integrating the tracker into your project with Carthage.
```
Comment (goodits) [inline _posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md]:
```suggestion
We are pleased to announce a new release of the [Snowplow Objective-C Tracker][ios-tracker]. [Version 1.0.0][tracker-100] brings dependency updates, bug fixes and support for integrating the tracker into your project with Carthage.
```

Comment (goodits) [reply _posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md]: Really minor and just trying to keep it concise.

File: _posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+{% highlight bash %}
+# Snowplow tracker
+github "snowplow/snowplow-objc-tracker"
+{% endhighlight %}
+
+<h2><a name="help">4. Getting help</a></h2>
+
+As always, please check out the following links:
+
+* The [technical documentation][tech-docs]
```
Comment (goodits) [inline _posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md]: Does this release not include the documentation refresh Mike?

Comment (mhadam) [reply _posts/2019-02-20-snowplow-objc-tracker-v1.0.0-released/index.md]: Updated already with highlights on the documentation

**Pull request merged**